### PR TITLE
Further normalize PeopleController and fix #311

### LINKED ISSUE #311: Notes should be listed from most recent to less recent
Currently notes are listed with the oldest note at the top.

We should list the notes from the most recent, to the least recent (using the `created_at` field).

### PR DESCRIPTION
**Pull request by yamakadi**
* Also cleans up Contact a bit more

* Merge remote-tracking branch 'upstream/master' into more_work_on_people_controller

* Fix oversight

### CODE REVIEW

**Comment by yamakadi** [comment]
Squashing the commits had some weird effects on Changelog and ImportCSV command, but I think it can still be merged.

File: app/Http/Controllers/PeopleController.php [hunk 1]
```diff
@@ -281,116 +257,77 @@ public function delete(Request $request, $contactId)
...
-            $contact->job = null;
-        }
+        $contact->job = ! empty($job) ? $job : null;
 
-        if ($company != '') {
-            $contact->company = $company;
-        } else {
-            $contact->company = null;
-        }
+        // There is no company field in the database.
```
Comment (djaiss) [inline app/Http/Controllers/PeopleController.php]: There actually is. Check https://github.com/monicahq/monica/blob/master/database/migrations/2017_06_13_195740_add_company_to_contacts.php.

**Comment by yamakadi** [comment]
I must not have run the migration. Sorry ><
I'll fix it in a bit

File: app/Http/Controllers/PeopleController.php (hunk 1 above)
Comment (yamakadi) [inline app/Http/Controllers/PeopleController.php]: Changed

**Comment by yamakadi** [comment]
@djaiss Are there anymore changes needed?

**Comment by djaiss** [comment]
@yamakadi excuse me, I was super busy on a huge PR in the last couple of days. Let me check this!

**Comment by djaiss** [comment]
Thanks a lot for your work!

**Pull request merged**